pos/src/main/java/edu/illinois/cs/cogcomp/pos/FeatureVectorParser.java

```diff
@@ -85,23 +89,23 @@ public FeatureVectorParser(String exampleFile, String lexiconFile) {
         try {
             lexIn = new ObjectInputStream(new FileInputStream(lexiconFile));
         } catch (Exception e) {
-            System.err.println("Can't open '" + lexiconFile + "' for input:");
```

Comment:
where the print is followed by a System.exit(), please print the error to STDERR.  @danyaljj do you agree?